airbyte-integrations/connectors/source-microsoft-onedrive/acceptance-test-config.yml

```diff
@@ -0,0 +1,31 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-microsoft-onedrive:dev
```

Comment:
we do have tests on that level, but as demonstrated with the bug a found (https://github.com/airbytehq/airbyte/pull/32655#discussion_r1418842805), it's possible to break this on the level of the individual connector quite easily, so I would feel much more comfortable with this additional layer of security.

My intent is that if someone comes along later and tries to refactor open_file, this kind of problem will be caught right away instead of by the user.

Reply:
added as second stream